sdk/identity/ci.yml

```diff
@@ -29,7 +29,7 @@ stages:
       CtestRegex: azure-identity.
       LiveTestCtestRegex: azure-identity.
       LineCoverageTarget: 99
-      BranchCoverageTarget: 63
+      BranchCoverageTarget: 62
```

Comment:
As the area owner, this is your judgement call (and you know best about coverage/feasibility of testing and organization for this PR). However, I caution against dropping code coverage number baselines whenever possible.